crates/forge/bin/cmd/build.rs

```diff
@@ -69,6 +69,12 @@ pub struct BuildArgs {
     #[clap(flatten)]
     #[serde(skip)]
     pub watch: WatchArgs,
+
+    ///Output the compilation errors in the json format.
+    /// This is useful when you want to use the output in other tools.
+    #[clap(long)]
```

Comment:
this should conflict with `silite`, "conflicts_with = "silent"" should work I believe, but better add a parser test as well:

ref https://github.com/foundry-rs/foundry/blob/e26f4d8585cc543dea8aa719ab89426fc17fcdaa/crates/forge/bin/cmd/build.rs#L151-L151